CPT331.iOS/CPT331.iOS/EventField.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  EventField.swift
+//  CPT331.iOS
+//
+//  Created by Peter Weller on 22/09/2016.
+//  Copyright © 2016 Peter Weller. All rights reserved.
+//
+
+import Foundation
+
+enum EventField:String {
```

Comment:
Are you able to expand on what this enum does? Generally, enums are used to define a list of permissible states rather than as a signal of an entity's type.